src/main/scala/com/fulcrumgenomics/bam/EstimatePoolingFractions.scala

```diff
@@ -134,16 +143,13 @@ class EstimatePoolingFractions
   }
 
   /** Verify a provided sample list, or if none provided retrieve the set of samples from the VCF. */
-  private def pickSamplesToUse(vcfReader: VCFFileReader): Array[String] = {
-    if (samples.nonEmpty) {
-      val samplesInVcf = vcfReader.getFileHeader.getSampleNamesInOrder.iterator.toSet
-      val missingSamples = samples.filterNot(samplesInVcf.contains)
+  private def pickSamplesToUse(vcfIn: VcfSource): Array[String] = {
+    if (this.samples.isEmpty) vcfIn.header.samples.toArray else {
+      val samplesInVcf   = vcfIn.header.samples
+      val missingSamples = samples.toSet.diff(samplesInVcf.toSet)
```

Comment:
I prefer the old implementation as it's clearer that it's samples specified on the command line that are missing in the VCF header.  `diff` in `Set` has always bugged me that it's directional.

Reply:
I mean, set difference is always a directional, I feel like this is like complaining that `minus` is directional for integers.  Still, maybe it would be clearer as `samples -- samplesInVcf`?

Reply:
Ha!

But why did you change it? Is more performant?

Reply:
`x.diff(y)` == `x -- y`.  I just figured the latter would be more to your liking.